Doc/library/queue.rst

```diff
@@ -128,6 +128,15 @@ provide the public methods described below.
    Otherwise (*block* is false), return an item if one is immediately available,
    else raise the :exc:`Empty` exception (*timeout* is ignored in that case).
 
+   .. warning::
```

Comment:
Let's just add this as plain text.  The warning directive is typically saved for cases that are several or common.  Otherwise, it creates visual litter than makes the docs hard to read (the eye is first drawn to the warning boxes).